aemedge/blocks/offer-cards/offer-cards.css

```diff
@@ -0,0 +1,240 @@
+    padding: 17px;
+  }
+
+.offer-cards-content .text-cta-container {
+    margin: 1rem 0;
+}
+
+.offer-cards-content .headline p {
+    font-size: 1.125rem;
+    line-height: 1.11;
+    padding: 0;
+    font-weight: bold;
+}
+
+  .offer-cards-content h1 {
+    font-size: 30px;
+    line-height: 1.2;
+    font-weight: 600;
+    letter-spacing: -1.15px;
+    margin: 0 0 1rem;
+    max-width: 600px;
+  }
+
+  .offer-cards-content h2 {
+    font-size: var(--heading-font-size-3xl);
+    line-height: 54px;
+    letter-spacing: -1.15px;
+    margin: 0 0 1rem;
+    max-width: 600px;
+    -webkit-font-smoothing: antialiased;
+    text-wrap: wrap;
+    scroll-margin: 0;
+  }
+
+  .offer-cards .background {
+    position:absolute;
+
+  }
+
+  .offer-cards .background picture img{
+    position: relative;
+    z-index: 0;
+    object-fit: cover;
+    width: 100%;
+    box-sizing: border-box;
+    inset: 0;
+      height: 100%;
+  }
+
+.offer-cards .background picture{
+    display: unset;
+}
+
+.offer-cards-content .sub-headline p {
+    font-size: 0.875rem;
+    line-height: 1.25;
+    padding: 0;
+    margin: 0 0 1rem;
+}
+
+  .offer-cards-content .text-cta-container .button-container
+  {
+    display: flex;
+    flex-direction: column;
+    -webkit-box-align: center;
+    align-items: center;
+    width: fit-content;
+    max-width: unset;
+  }
+
+  .offer-cards-content .text-cta-container .button-container p {
+  margin:0;
+ }
+
+
+.offer-cards-content .foreground-container {
+  margin: 1rem auto;
+    text-align: center;
+}
+
+.offer-cards-content .badge {
+    line-height: 1;
+    vertical-align: bottom;
+    padding: 0 7px;
+    font-size: 0.875rem;
+    font-weight: 600;
+    background-color: rgb(16 6 159);
+    color: rgb(245 245 246);
+    border-radius: 0.125rem;
+    text-transform: uppercase;
+    margin: 0 0 1rem;
+    width: fit-content;
+}
+
+.offer-cards-container .default-content-wrapper .button-container {
+    margin-left: 114px;
+}
+
+.offer-cards-wrapper {
+    margin-bottom: 20px;
+}
+
+.offer-cards .backgroundlg{
+    border-radius: 0.25rem 0.25rem 0 0;
+    background: linear-gradient(transparent 0%, rgb(32 34 48 / 75%) 60%, rgb(32 34 48) 100%);
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: 100%;
+}
+
+@media (width >= 768px) {
+    .offer-cards.block {
+        box-shadow: rgb(7 7 11 / 40%) 0 4px 8px 0;
+    }
+
+    .offer-cards-content {
+        min-height: 366px;
+        padding: 35px 20px;
+    }
+
+    .offer-cards .background picture img{
+        height: 100%;
+    }
+
+    .section.offer-cards-container{
+        display: flex;
+        flex-wrap: wrap;
+        max-width: fit-content;
+        margin-left: auto;
+        margin-right: auto;
+        justify-content: center;
+    }
+
+    .offer-cards-container>div:not(.offer-cards-wrapper)
+    {
+        display: block;
+        width:100%
+    }
+
+    .section.offer-cards-container>div.offer-cards-wrapper{
+        padding: 5px;
```

Comment:
Change from padding to margin so that the hover effect doesn't show a white strip:
`margin: 5px`